tests/email.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from typing import List, Optional
+from uuid import UUID
+
+
+@dataclass
+class Email:
+    subject: str
+    recipients: List[str]
+    html: str
+    sender: str
+
+
+class FakeEmailSender:
+    def __init__(self) -> None:
+        self.sent_mails: List[Email] = []
+
+    def send_message(
+        self,
+        subject: str,
+        recipients: List[str],
+        html: str,
+        sender: str,
+    ) -> None:
+        self.sent_mails.append(
+            Email(
+                subject=subject,
+                recipients=recipients,
+                html=html,
+                sender=sender,
+            )
+        )
+
+
+@dataclass
```

Comment:
does this class have to be a `dataclass`?

Reply:
it was unnecessary. I removed the decorator.